_rules/image-no-text-0va7u6.md

```diff
@@ -29,20 +29,21 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any [visible][] [embedded image][] that is not [marked as decorative][].
+This rule applies to any [visible][] [embedded image][].
 
 ## Expectation
 
-For each of the image resources referenced by the [image sources][] of the test target, at least one of the following is true:
+For each test target, at least one of the following is true:
 
+- the image resource contains text and it is [essential][] that the text is rendered with that specific presentation; or
```

Comment:
We need to define what the image source is.

Reply:
This definition doesn't really work. I think I know what you're trying to say here, but the image source is a URL, not the actual image. I also don't think you can use canvas the way you're using it. What is the image source of a canvas? Even assuming someone can figure out whether or not it used drawImage to render; what if multiple drawImage calls were made?

Reply:
Would "rendered image resource" (as used in the first line of the expectation" solve this issue for you?

Regarding canvas, I'm going to drop it based on the difficulty that is finding out if drawImage is being used or not.